README.md

```diff
@@ -29,6 +29,7 @@ Legacy standard compatible
 - [Fewcha](https://www.npmjs.com/package/fewcha-plugin-wallet-adapter)
 - [Martian](https://www.npmjs.com/package/@martianwallet/aptos-wallet-adapter)
 - [MSafe](https://www.npmjs.com/package/@msafe/aptos-wallet-adapter)
+- [MizuWallet](https://www.npmjs.com/package/@mizuwallet-sdk/aptos-wallet-adapter)
```

Comment:
I believe MizuWallet is AIP-62 standard compatible? then it should be included in the section above

Reply:
actually, looking at the package - looks like you use the [legacy standard ](https://www.npmjs.com/package/@mizuwallet-sdk/aptos-wallet-adapter?activeTab=code) tho you also have the new standard implementation - any reason for that?

Reply:
The file for AIP-62 standard is backup. `PublicKey` is empty in multisign account v2 but required in new standard. We're going to implement the plugin to new standard once the `PublicKey` issue be resolved. And now, there's no entry for the new standard in the backup file.

Reply:
Updated.
For MizuWallet is compatible with AIP-62 standard.